### ISSUE TITLE
Reached the max retry count error

### ISSUE THREAD

**Issue by regadas** [xerial/sbt-sonatype#149]
Hi @xerial 

with the latest release I'm seeing this:

```
2020-03-27 13:10:01.257Z error [Sonatype] 
77
wvlet.airframe.http.HttpClientMaxRetryException: Reached the max retry count 0/0: [504: Gateway Timeout] Request failed: <html>
78
<head><title>504 Gateway Time-out</title></head>
79
<body bgcolor="white">
80
<center><h1>504 Gateway Time-out</h1></center>
81
</body>
82
</html>
```

Any new option I'm missing?

**Comment by Taig** [comment]
I'm also experiencing this, but the same is happening with the previous release (3.8.1). So I assume the issue is on sonatype's end and will be resolved eventually.

**Comment by plokhotnyuk** [comment]
Sonatype Ops are working on that: https://twitter.com/sonatype_ops/status/1243532338090381312

I hope they will update their status page for today: https://status.maven.org/#week

**Comment by regadas** [comment]
oh 3.8.1 worked somehow for me. Anyway, glad it's on sonatype! I'll close this once it's back to normal. (just in case somebody stumbles upon this as well)

**Comment by xerial** [comment]
Yeah. I also experienced the same issue. Sonatype OSS API is now under maintenance. For a relieve, I'll increase the total timeout (upto 10 min) for individual HTTP requests and release sbt-sonatype 3.9.2: https://github.com/xerial/sbt-sonatype/commit/24ab0e97bfa27b114021cca939d5b3fae48c62f5

**Comment by xerial** [comment]
Released sbt-sonatype 3.9.2. We still may see this error if sonatype is under maintenance or returns 504. In this case, we need to retry `sonatypeBundleRelease` after a while.
